x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -319,8 +319,6 @@
     "common.ui.stateManagement.unableToStoreHistoryInSessionErrorMessage": "セッションがいっぱいで安全に削除できるアイテムが見つからないため、Kibana は履歴アイテムを保存できません。\n\nこれは大抵新規タブに移動することで解決されますが、より大きな問題が原因である可能性もあります。このメッセージが定期的に表示される場合は、{gitHubIssuesUrl} で問題を報告してください。",
     "common.ui.url.replacementFailedErrorMessage": "置換に失敗、未解決の表現式: {expr}",
     "common.ui.url.savedObjectIsMissingNotificationMessage": "保存されたオブジェクトがありません",
-    "common.ui.vis.kibanaMap.leaflet.fitDataBoundsAriaLabel": "データバウンドを合わせる",
```

Comment:
why are these gone?

Reply:
I ran both an `i18n` check w/ fix flag and an extract. Neither of these indicated anything missing or out of place wrt `maps_legacy`. i18n did complain a few commits back about adding the new plugin directory, which I did [here](https://github.com/elastic/kibana/blob/4fc42acde34bb54ec3457e0128c77f558308a996/.i18nrc.json#L27). It's likely that some translations require manual translation and are flagged in a separate documentation process. Unfortunately it's not as easy as just updating the dot path to its new location since this would just be wiped out by the next `node scripts/i18n_extract` command someone runs. I did notice when running `node scripts/i18n_extract` that both of these paths were correctly grabbed for the english (untracked) file here:

![image](https://user-images.githubusercontent.com/1728976/79158208-286b3300-7d93-11ea-8911-93898153c547.png)

So they are tracked correctly and should be translated in a later process